example-rest-api-integration/src/api/api.service.ts

```diff
@@ -0,0 +1,43 @@
+import { AppConfig } from 'src/configuration/app.config';
+import { Injectable } from '@nestjs/common';
+import { Profile } from './model/profile.model';
+import axios from 'axios';
+
+interface LucidHeaders {
+  'Lucid-Api-Version': string;
+  Authorization: string;
+}
+
+@Injectable()
+export class ApiService {
+  constructor(private readonly config: AppConfig) {}
+
+  /**
+   * This method performs the Get Profile request as described in https://developer.lucid.co/rest-api/v1/#get-profile86
```

Comment:
```suggestion
   * This method performs the Get Profile request as described in https://developer.lucid.co/rest-api/v1/#get-profile
```
I don't know why it appends the 86, but it works without it and looks a little more professional